src/modules/number/index.ts

```diff
@@ -84,22 +85,26 @@ export class NumberModule extends SimpleModuleBase {
   }
 
   /**
-   * Returns a single random floating-point number for a given precision or range and precision.
-   * The lower bound is inclusive, the upper bound is exclusive, unless precision is passed.
+   * Returns a single random floating-point number for a given multipleOf or range and multipleOf.
+   * The lower bound is inclusive, the upper bound is exclusive, unless multipleOf is passed.
    *
    * @param options Upper bound or options object.
    * @param options.min Lower bound for generated number. Defaults to `0.0`.
    * @param options.max Upper bound for generated number. Defaults to `1.0`.
    * @param options.precision Precision of the generated number, for example `0.01` will round to 2 decimal points.
    * If precision is passed, the upper bound is inclusive.
+   * @param options.multipleOf The generated number will be a multiple of this property.
+   * This property can be used to achieve specific decimal points.
```

Comment:
```suggestion
   * This property can be used to limit the result a specific number of decimal digits.
```

(I have never used/heart of `decimal points`, but that might just be me not being familiar with it)